src/translator/matrix.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <vector>
+
```

Comment:
Could we please add just a small documentation regarding what this class does?

Reply:
@abarbosa94 The problem here is this matrix gives a perception of being top-level despite not truly being so. This matrix is not general enough to be used outside LogisticRegressor based QE. Restrict availability and usage to where it sticks properly (public in LR for use in LRQE and unit tests?). It should be evident from the placement that this class is not for external consumption. Documentation need only be minimal, discussing how it pertains to usage in LRQE.

That should get us to -2 files as well.

Reply:
Documentation process was completed( Commit 2f81645 ) and we have moved the matrix class to LRQE. Commit dda4c80